clang/lib/CIR/Dialect/Transforms/TargetLowering/CIRLowerContext.h

```diff
@@ -42,7 +43,7 @@ class CIRLowerContext : public llvm::RefCountedBase<CIRLowerContext> {
 
   /// The language options used to create the AST associated with
   /// this ASTContext object.
-  clang::LangOptions &LangOpts;
+  clang::LangOptions LangOpts;
```

Comment:
Why do we now need to make a new copy of this? I'd expect langoptions to survive til the end.

Reply:
Someone has to retain ownership of LangOpts to prevent dangling refs. `CIRLowerContext` is the owner in this case.